src/utils/rest-api.js

```diff
@@ -55,7 +55,12 @@ export function connectNotificationsWsUpdateTask() {
     const webSocketUrl =
         webSocketBaseUrl + PREFIX_TASK_NOTIFICATION_WS + '/websocket';
 
-    const reconnectingWebSocket = new ReconnectingWebSocket(webSocketUrl);
+    let webSocketUrlWithToken;
+    webSocketUrlWithToken = webSocketUrl + '&access_token=' + getToken();
```

Comment:
Can't you merge declaration and assignment ?

Reply:
done